examples/drush

```diff
@@ -1,13 +1,32 @@
 #!/usr/bin/env sh
 #
-# A wrapper script which launches the Drush that is in your project's /vendor directory.
-# Copy it to the root of your project (or wherever you store handy scripts) and edit as desired.
+# DRUSH WRAPPER
 #
-# Below are options which you might want to add. More info at `drush topic core-global-options`:
+# A wrapper script which launches the Drush that is in your project's /vendor
+# directory.  Copy it to the root of your project (or wherever you store handy
+# scripts) and edit as desired.  If you already have a folder named 'drush',
```

Comment:
Yes, but if you put it in __ROOT__/vendor/bin, and you have added that to your $PATH, then it will work.  Maybe that's too complicated, and we should just remove this advice, though?